tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -187,12 +198,12 @@ dnl mclk_id is optional
 dnl ssp1-maxmspk, ssp0-RTHeadset
 
 #SSP 1 (ID: 7)
-DAI_CONFIG(SSP, 1, 7, SSP1-Codec,
-	SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, 38400000, codec_mclk_in),
-		      SSP_CLOCK(bclk, 9600000, codec_slave),
-		      SSP_CLOCK(fsync, 48000, codec_slave),
-		      SSP_TDM(8, 25, 3, 3),
-		      SSP_CONFIG_DATA(SSP, 1, 24)))
+#DAI_CONFIG(SSP, 1, 7, SSP1-Codec,
+#	SSP_CONFIG(DSP_B, SSP_CLOCK(mclk, 38400000, codec_mclk_in),
```

Comment:
Still needed ?

Reply:
It is not needed, I remember had deleted them before pushing, maybe a mistake there.

Thanks, updating.